src/main/java/kdt/springbootboardjpa/service/dto/PostDto.java

```diff
@@ -0,0 +1,13 @@
+package kdt.springbootboardjpa.service.dto;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.PositiveOrZero;
+import lombok.Builder;
+
+public record PostDto(Long id, @NotBlank(message = "NotBlank") String title, @NotBlank(message = "NotBlank") String content, @Positive(message = "Positive") Long createdBy) {
```

Comment:
```suggestion
public record PostDto(
	Long id,
	String title,
	String content,
	Long createdBy
) {
	//...
}
```

파라미터가 많아져서 길어진다면 아래 기능을 활용해주세요!

<img width="651" alt="CleanShot 2023-01-03 at 11 02 19@2x" src="https://user-images.githubusercontent.com/17922700/210291366-ae530132-7656-465c-a40e-21d70d3ec743.png">